src/main/kotlin/com/coder/toolbox/util/LinkMap.kt

```diff
@@ -11,7 +11,7 @@ private const val FOLDER = "folder"
 private const val IDE_DOWNLOAD_LINK = "ide_download_link"
 private const val IDE_PRODUCT_CODE = "ide_product_code"
 private const val IDE_BUILD_NUMBER = "ide_build_number"
-private const val IDE_PATH_ON_HOST = "ide_path_on_host"
+private const val IDE_PATH_ON_HOST = "project_path"
```

Comment:
How is `IDE_PATH_ON_HOST` the same as `project_path`? Probably a naming confusion only.

Reply:
Done. Sorry about that, brainfart on my side :)